hastexo/public/js/main.js

```diff
@@ -270,6 +273,52 @@ function HastexoXBlock(runtime, element, configuration) {
         });
     };
 
+    var construct_layout = function() {
+        var instructions_layout = configuration.instructions_layout;
+
+        /* 'above' is the default layout and doesn't require any changes */
+        if (instructions_layout != 'above') {
+            var content = document.getElementsByClassName("vert-mod")[0];
+
+            var parent_element = function(child_element) {
+                for (var div of content.children)
+                    if (div.contains(child_element)) {
+                        return div;
+                    };
+            };
+            var instructions = document.getElementsByClassName('lab_instructions')[0];
+
+            var terminal_parent = parent_element(document.getElementById('terminal'));
+            var instructions_parent = parent_element(instructions);
+
+            if (terminal_parent != 'undefined' && instructions_parent != 'undefined') {
```

Comment:
It looks like the condition in parentheses here should always be true. Do you perhaps want an `else` block here that writes a warning to the JavaScript console when it's false, to catch logic bugs?